wp-content/themes/twentyseventeen-wp20/template-parts/header/navigation-top.php

```diff
@@ -3,11 +3,13 @@
  * Displays top navigation
  */
 
+$title = is_single() ? __( 'News', 'wp20' ) : get_the_title();
+?>
 
-?>	<button class="menu-toggle" aria-controls="top-menu" aria-expanded="false">
+<button class="menu-toggle" aria-controls="top-menu" aria-expanded="false">
 		<img class="icon-bars" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/images/menu-icon.svg" aria-hidden="true"  />
 		<img class="icon-close" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/images/close-icon.svg" aria-hidden="true" />
-		<span><?php esc_html_e( get_the_title() ); ?></span>
+		<span><?php esc_html_e( $title ); ?></span>
```

Comment:
```suggestion
		<span><?php echo esc_html( $title ); ?></span>
```

The translation functions can't work with variables, only raw strings.